addon/package.json

```diff
@@ -66,13 +75,25 @@
   "engines": {
```

Comment:
now this field can be removed as there is nothing about Node.js for v2 addons

Reply:
Do we not need to install node in ci.yml then?

Reply:
we indeed Node.js to precompile addon and run tests in test-app, it's just now we could bump node version in ci.yml without need to do a major version bump if that's what you meant.